source/main.js

```diff
@@ -49,74 +55,121 @@ const MARKDOWN = {
   '<sup>': '^',
   '</sup>': '',
   '	': '', // special tab
+  '<span.*?>': '',
+  '</span>': '',
 };
 
 const copyText = (isMarkdown) => {
+  // Get the current URL.
   const url = window.location.href;
-  const title = document.querySelector('[data-cy=question-title]').innerText;
-  const text = Array.from(
-    document.querySelector('[data-key=description-content]').children
-  )[0].children[1].innerText;
 
-  const hiddenElement = document.createElement('textarea');
+  // Try to find the elements for the old version of the website.
+  let title;
+  let descriptionContent;
+  let text;
+  let html;
+  try {
+    title = document.querySelector('[data-cy=question-title]').innerText;
+    descriptionContent = Array.from(
+      document.querySelector('[data-key=description-content]').children
+    )[0].children[1];
+    text = descriptionContent.textContent.replace(/(\n){2,}/g, '\n\n').trim();
+    html = descriptionContent.innerHTML;
+    if (text == null || html == null) {
+      throw "Old version elements not found";
+    }
+  } catch (err) {
+    // If the elements for the old version are not found, try finding the elements for the new version.
+    title = document.querySelector('.mr-2.text-lg.font-medium.text-label-1.dark\\:text-dark-label-1').innerText;
```

Comment:
it looks like this targets only the element in dark mode

Reply:
Thanks for checking it out. But it looks like the element names aren't actually connected to the theme color. I don't know why LeetCode named them that way. At first, I also thought they were.

**As you can see it works on light mode also:**
![image](https://user-images.githubusercontent.com/85361211/214172260-96b0e9af-d963-4dbe-9db8-17e03bf33429.png)

**Markdown generated for above problem in light mode:**

# [58. Length of Last Word](https://leetcode.com/problems/length-of-last-word/)

Given a string `s` consisting of words and spaces, return the length of the **last**  word in the string.

A **word**  is a maximal substring consisting of non-space characters only.

**Example 1:** 

```
Input: s = "Hello World"
Output: 5
Explanation: The last word is "World" with length 5.

```

**Example 2:** 

```
Input: s = "   fly me   to   the moon  "
Output: 4
Explanation: The last word is "moon" with length 4.

```

**Example 3:** 

```
Input: s = "luffy is still joyboy"
Output: 6
Explanation: The last word is "joyboy" with length 6.

```

**Constraints:** 

- `1 <= s.length <= 10^4`
- `s` consists of only English letters and spaces `' '`.
- There will be at least one word in `s`.